scripts/run_docker_build.sh

```diff
@@ -18,14 +18,20 @@ CONDARC
 )
 
 cat << EOF | docker run -i \
-                        -v ${REPO_ROOT}/recipes:/conda-recipes \
+	                -v ${REPO_ROOT}:/staged-recipes \
                         -a stdin -a stdout -a stderr \
                         $IMAGE_NAME \
                         bash || exit $?
 
 # Copy the host recipes folder so we don't ever muck with it
-cp -r /conda-recipes /conda-recipes-without-example
-rm -rf /conda-recipes-without-example/example
+cp -r /staged-recipes/recipes /conda-recipes
+
+# Find the recipes from master in this PR and remove them.
+echo "Finding recipes merged in master and removing them from the build."
+pushd /staged-recipes/recipes > /dev/null
+git fetch https://github.com/conda-forge/staged-recipes.git master
```

Comment:
Though should add this general CircleCI strategy may fail if `master` has already converted those recipes. This is because CircleCI does not merge with `master`, but works on the PR tip. To fix that case, we can actually just merge with `master` ourselves, which will clear the converted recipes from our history automatically. PR ( https://github.com/conda-forge/staged-recipes/pull/1486 ) will do exactly that.